tests.py

```diff
@@ -208,3 +208,81 @@ def test_sip_api_already_set():
             import sip
             sip.setapi("QString", 1)
             assert_raises(ImportError, __import__, "Qt")
+
+
+def test_translate_and_UnicodeUTF8_in_pyside():
+    with pyside():
+        from Qt import QtWidgets
+
+        # this does exist in PySide
+        assert QtWidgets.QApplication.UnicodeUTF8 is not None
+
+        # use patched method with old arguments
+        QtWidgets.QApplication.translate(
```

Comment:
Sure thing, just wanted to check the overall testing first (which is failing).
I'll add a check of the return value once I have something basic working ;)